.github/pipelines/github-semmle.yml

```
@@ -37,6 +37,10 @@ extends:
             steps:
               - checkout: self
                 clean: true
+              - task: UseDotNet@2
+                displayName: Temporarily install .NET 8 SDK while dual targeting
+                inputs:
+                  version: "8.0.404"
```

Feel free to ignore but using N.x is a better approach for the lower version, so it doesn't need to be manually bumped as security patches are released.
```suggestion
                  version: "8.x"
```